config/superblocks.ts

```diff
@@ -169,6 +169,24 @@ export const notAuditedSuperBlocks: NotAuditedSuperBlocks = {
     SuperBlocks.JsAlgoDataStructNew,
     SuperBlocks.TheOdinProject,
     SuperBlocks.FoundationalCSharp
+  ],
+  [Languages.Swahili]: [
+    SuperBlocks.DataVis,
+    SuperBlocks.RelationalDb,
+    SuperBlocks.BackEndDevApis,
+    SuperBlocks.QualityAssurance,
+    SuperBlocks.SciCompPy,
+    SuperBlocks.DataAnalysisPy,
+    SuperBlocks.InfoSec,
+    SuperBlocks.MachineLearningPy,
+    SuperBlocks.CollegeAlgebraPy,
+    SuperBlocks.CodingInterviewPrep,
+    SuperBlocks.ProjectEuler,
+    SuperBlocks.TheOdinProject,
+    SuperBlocks.FoundationalCSharp,
+    SuperBlocks.RespWebDesign,
+    SuperBlocks.JsAlgoDataStruct,
+    SuperBlocks.FrontEndDevLibs
```

Comment:
```suggestion
    SuperBlocks.FrontEndDevLibs,
    SuperBlocks.JsAlgoDataStructNew
```

This allows builds to succeed with `SHOW_UPCOMING_CHANGES`.

Reply:
Changes have been updated. Kindly confirm if its correctly done